pkg/faces/constants.go

```diff
@@ -17,42 +17,102 @@
 
 package faces
 
-var Smileys = map[string]string{
-	"Smiling":     "&#x1F603;",
-	"Sleeping":    "&#x1F634;",
-	"Cursing":     "&#x1F92C;",
-	"Kaboom":      "&#x1F92F;",
-	"HeartEyes":   "&#x1F60D;",
-	"Neutral":     "&#x1F610;",
-	"RollingEyes": "&#x1F644;",
-	"Screaming":   "&#x1F631;",
+type SmileyMap struct {
+	smileys map[string]string
 }
 
-// colorblind-friendly colors from the Tol palette
-var Colors = map[string]string{
-	"grey":    "grey",
-	"purple":  "rgb(48 34 130)",
-	"green":   "rgb(55 117 59)",
-	"cyan":    "rgb(55 117 59)",   // too similar to pink and grey, avoid
-	"blue":    "rgb(151 202 234)", // light blue
-	"yellow":  "rgb(218 204 130)",
-	"pink":    "rgb(191 108 120)",
-	"hotpink": "rgb(158 75 149)",
-	"red":     "rgb(125 42 83)", // dark magenta
```

Comment:
Check for places elsewhere where some of these older names were used, specifically:
- [ ] `purple` -> `violet`
- [ ] `cyan` -> `bluegreen`
- [ ] `pink` -> `coral`
- [ ] `hotpink` -> `fuschia`